src/intf/panic.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include <stdint.h>
+#include <stddef.h>
+
+/*
+vcm 08/09/2021
+Maybe make this private so no virus makers can use Panic in their viruses 
+
+*/
```

Comment:
this can be discussed in the discussions section or in issues, not necessary here